PATHFIND/GetClosestRoad.md

```diff
@@ -5,7 +5,7 @@ ns: PATHFIND
 
 ```c
 // 0x132F52BBA570FE92 0x567B0E11
-Any GET_CLOSEST_ROAD(float x, float y, float z, float p3, int p4, Vector3* p5, Vector3* p6, Any* p7, Any* p8, float* p9, BOOL p10);
+int GET_CLOSEST_ROAD(float x, float y, float z, float p3, int n, Vector3* srcNode, Vector3* destNode, int* forwardLanes, int* backwardLanes, float* width, BOOL onlyMajorRoads);
```

Comment:
Anywhere you changed types also needs to has cs_type(PreviousType), as an example for p7 `cs_type(Any*) int* forwardLanes`

Reply:
Including the return type of the native, yeah?

Will fix